oauth2client/client.py

```diff
@@ -469,6 +467,11 @@ def clean_headers(headers):
     return clean
 
 
+def _parse_expiry(expiry):
+    if expiry and isinstance(expiry, datetime.datetime):
+        return expiry.strftime(EXPIRY_FORMAT)
```

Comment:
@nathanielmanistaatgoogle This seems to be an arbitrary recommendation (that also happens to end up with more code)